contrib/numpy/template.py

```diff
@@ -0,0 +1,31 @@
+pkgname = "numpy"
```

Comment:
python-numpy

this is also pretty useless without fortran and openblas. you should put this in a separate mr and we can work on those separately

also, i forget if python_pep517 even passes through cflags and stuff, does it?

Reply:
also the versions here are input wrong etc